crates/figma_import/src/serialized_document.rs

```diff
@@ -52,3 +77,129 @@ pub struct ServerFigmaDoc {
     pub project_files: Vec<FigmaDocInfo>,
     pub errors: Vec<String>,
 }
+
+/// A helper method to load a serialized design doc from figma.
+pub fn load_design_doc<P>(
+    load_path: P,
+) -> Result<(SerializedDesignDocHeader, SerializedDesignDoc), Error>
+where
+    P: AsRef<Path>,
+{
+    let mut document_file = match File::open(&load_path) {
+        Ok(file) => file,
+        Err(e) => {
+            return Err(e.into());
+        }
+    };
+
+    let mut buf: Vec<u8> = vec![];
+    let _bytes = document_file.read_to_end(&mut buf)?;
+
+    let header: SerializedDesignDocHeader = match bincode::deserialize(buf.as_slice()) {
```

Comment:
I think this could be `let header: SerializedDesignDocHeader = bincode::deserialize(buf.as_slice())?;`, and similarly you could do this with other places in these functions that return a Result.